examples/twittermap/noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/FeedSocketAdapterClient.java

```diff
@@ -16,14 +16,16 @@
     protected int maxCount;
 
     protected Socket socket;
+    protected int socketNumber;
 
     public FeedSocketAdapterClient(String adapterUrl, int port, int batchSize,
-                                         int waitMillSecPerRecord, int maxCount) {
+                                   int waitMillSecPerRecord, int maxCount, int socketNumber) {
```

Comment:
if `socketNumber` is only used for logging, why not directly using the `adapterUrl`? which contains more information and straightforward. Moreover, we don't even need this parameter.

Reply:
Sure.